index.html

```diff
@@ -13,7 +14,81 @@
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
+  <body>
+
+    <div class="container">
+
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="product.html">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+        </nav>
+    
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </header>
+    
+      <section class="cta">
+        <div class="cta-text">
+          <h1>Innovation<br> On<br> Demand</h1>
+          <button>Get Started</button>
+        </div>
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </section>
+    
+      <section class="main-content">
+    
+        <div class="top-content">
+          <div class="text-content">
+            <h4>Features</h4>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="text-content">
+            <h4>About</h4>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </div>
+    
+        <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    
+        <div class="bottom-content">
+          <div class="text-content">
+            <h4>Services</h4>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="text-content">
+            <h4>Product</h4>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="text-content">
+            <h4>Vision</h4>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </div>
+    
+      </section>
+    
+      <section class="contact">
+    
+        <h4>Contact</h4>
+    
+        <p>123 Way 456 Street<br> Somewhere, USA</p>
+    
+        <p>1 (888) 888-8888</p>
+    
+        <p>sales@greatidea.io</p>
+    
+      </section>
+    
```

Comment:
```suggestion
        <h4>Contact</h4>

        <p>123 Way 456 Street<br> Somewhere, USA</p>

        <p>1 (888) 888-8888</p>

        <p>sales@greatidea.io</p>
```

Consider placing this section inside an `<address>` tag for better semantic HTML and search engine optimization.